acrobat/blocks/verb-widget/verb-widget.js

```diff
@@ -701,57 +703,26 @@ export default async function init(element) {
 
     const analyticsMap = {
       change: () => {
-        window.analytics.verbAnalytics('choose-file:open', VERB, mergeData({ ...data, userAttempts }));
+        const metadata = mergeData({ ...data, userAttempts });
```

Comment:
This metadata is defined in multiple places. Can we define above in one spot so it can be reused?